c_glib/arrow-glib/reader.cpp

```diff
@@ -801,6 +801,92 @@ garrow_feather_file_reader_get_columns(GArrowFeatherFileReader *reader,
   return g_list_reverse(columns);
 }
 
+/**
+ * garrow_feather_file_reader_read:
+ * @reader: A #GArrowFeatherFileReader.
+ * @error: (nullable): Return locatipcn for a #GError or %NULL.
```

Comment:
typo? `locatipcn` => `location`
It seems to be using `locatipcn` in several places.

Reply:
Good catch!